movi/simulator/services/async_routing_service.py

```diff
@@ -0,0 +1,31 @@
+"""Modified RoutingService.route to accept od_pairs list and make asynchronous requests to it"""
+
+from osrm_router import RouteRequester
+from async_requests import AsyncRequest
+import json
+
+class RoutingService(object):
```

Comment:
shoud be a different class name? e.g., OSRMRoutingService

Reply:
Changed to AsyncRoutingService